docker/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM ubuntu:18.04
+MAINTAINER Bo Zhang (zhangb1@email.chop.edu)
```

Comment:
According to [Official Docker Documentation](https://docs.docker.com/engine/reference/builder/#copy) the `MAINTAINER` instruction is deprecated

https://riptutorial.com/docker/example/11010/maintainer-instruction#:~:text=According%20to%20Official%20Docker%20Documentation,with%20the%20command%20docker%20inspect%20.

Reply:
I will change that